imports/server/discordClientRefresher.ts

```diff
@@ -113,7 +113,7 @@ class DiscordClientRefresher {
             // If we get the lock, we're responsible for opening the websocket
             // gateway connection
             const ready = new Promise<void>((r) => client.on('ready', r));
-            client.login(this.token);
+            Promise.await(client.login(this.token));
```

Comment:
That's another correctness fix, nice

Reply:
The `await` on `ready` on the next line will effectively incorporate what this is `await`ing on, so I think it's technically not a correctness fix, but I figured it was safer to use the `void` cast throwaway sparingly when it wasn't necessary.